MatrixSDK/Utils/MXHTTPClient.m

```diff
@@ -649,6 +651,27 @@ - (void)setRequestParametersInJSON:(BOOL)requestParametersInJSON
     {
         httpManager.requestSerializer = [AFHTTPRequestSerializer serializer];
     }
+
+    // Overwrite user agent to include concrete device model (code copied from AlamoFire)
```

Comment:
The code below is copied from `AFHTTPRequestSerializer`. I decided to keep this in the ObjC part because this way it's an exact copy and possibly easier to maintain than a Swift port.